src/plugins/send_events.mjs

```diff
@@ -0,0 +1,44 @@
+import { Socket } from 'net';
+import path from 'path';
+
+const appJsonFilename = 'app.json';
+const packagerConnectPort = 9092;
+
+const packagerConnect = async (rootPath) => {
+  const appJsonPath = path.join(rootPath, appJsonFilename);
+  // const appConfig = await import(appJsonPath);
+  const { default: appConfig } = await import(appJsonPath, { assert: { type: "json" } });
+
+  if (!appConfig.remoteIP) {
+    throw new Error(appJsonFilename + ' is missing remoteIP field');
+  }
+
+  const client = new Socket();
+  client.connect(packagerConnectPort, appConfig.remoteIP, () => {
+    console.log('Connected to Sleeper App at ', appConfig.remoteIP);
+
+    client.setEncoding('utf8');
+    const json = JSON.stringify({
+      _webpack: 'packager_connect',
+      _name: appConfig.name ?? 'please_set_name',
```

Comment:
Minor, but can we leave this blank if no name is specified? I believe there's a bunch of module federation stuff that relies on this name, so it would likely be broken with the placeholder.